src/plugins/vis_type_timeseries/server/usage_collector/get_usage_collector.ts

```diff
@@ -59,10 +63,21 @@ export const getStats = async (
       // invalid visState
     }
 
-    if (
-      visState.type === 'metrics' &&
-      visState.params.time_range_mode === TIME_RANGE_DATA_MODES.LAST_VALUE
-    ) {
+    if (visState.type === 'metrics') {
+      timeseriesEmbeddables.push(visState);
+    }
+  }
+
+  const byValueVisualizations = await findByValueEmbeddables(soClient, 'visualization');
+
+  for (const item of byValueVisualizations) {
+    if ((item.savedVis as { type: string }).type === 'metrics') {
+      timeseriesEmbeddables.push(item.savedVis as VisState);
```

Comment:
I understand why you did it but pushing `byValueVisualizations` into `timeseriesEmbeddables` looks difficult for understanding

Reply:
I think you can create a simple method e.g. `telemetryUseLastValueMode` and use it instead of  pushing into `timeseriesEmbeddables` array

Reply:
Agree. Done.